src/main/java/com/transferwise/banks/demo/client/ErrorHandler.java

```diff
@@ -11,13 +11,19 @@
 @RestControllerAdvice
 public class ErrorHandler {
 
+    //@ExceptionHandler(ResponseStatusException.class)
+    public ResponseEntity<String> handleCustomException(final ResponseStatusException cause) {
+        return new ResponseEntity(new ApiError(cause), cause.getStatus());
+    }
+
     @ExceptionHandler(WebClientResponseException.class)
     public ResponseEntity<ApiError> handleWebClientResponseException(final WebClientResponseException ex) {
         return new ResponseEntity(new ApiError(ex), ex.getStatusCode());
+        //return ResponseEntity.status(ex.getRawStatusCode()).body(ex.getResponseBodyAsString());
```

Comment:
we should delete this line if it's not in use

Reply:
That was already there but I can remove it with the other changes.

Reply:
weird...yes plz remove